jekyll-assets/css/style.css

```diff
@@ -1,35 +1,71 @@
 :root {
-  --red: #cd2355;
+  --bg: #000000;
+  --footer-bg: #050609;
+  --near-bg: #111;
+  --top-title-colour: #343434;
   --red-tint: #d75a64;
-  --docsearch-primary-color: var(--red);
+  --brand-colour: #cd2355;
+  --copy-button-colour: #d8d8d8;
+  --code-bg-colour: #151515;
+  --toc-hover-colour: #252525;
+  --subtle: #707070;
+  --less-subtle: #333;
+  --home: #cd2355;
+  --rp2040-context-tag: #50C878;
+  --accent: #cd2355;
+  --docsearch-primary-color: var(--accent);
   --docsearch-logo-color: var(--red-tint);
+  --textcolor: white;
+  --subtle-text: #CCC;
+  --theme-toggle-label: "☀️";
+  --not-expanded-label: "▶";
+  --expanded-label: "▼";
+  color: var(--textcolor);
+}
+.light {
+  --bg: #fff;
+  --footer-bg: #f5f6f9;
+  --near-bg: #f6f6f6;
+  --red-tint: #d75a64;
+  --top-title-colour: #343434;
+  --brand-colour: #cd2355;
```

Comment:
Please forgive me if this is a stupid question, but if this `--brand-colour` value is identical in both the `:root` and `.light` sections, does it really need to be specified in two separate places? (and obviously also any other colours that are the "same" in both sections)

Reply:
Not a stupid question. Actually a stupid implementation from me. Part of my aforementioned consolidation of variables will be a 'common' non-theme-dependent set of vars.